CHANGELOG/CHANGELOG-0.2.md

```diff
@@ -2,8 +2,27 @@
 
 Changes since `v0.1.0`:
 
-- Fixed bug in a BestEffortFIFO ClusterQueue where a workload might not be
-  retried after a transient error.
-- Bumped the API version from v1alpha1 to v1alpha2. v1alpha1 is no longer supported and Queue is now named LocalQueue.
+### Features
+
+- Upgrade the API version from v1alpha1 to v1alpha2. v1alpha1 is no longer supported.
+- Rename Queue to LocalQueue.
+- Add webhooks to validate and to add defaults to all kueue APIs.
+- Support [codependent resources](/docs/concepts/cluster_queue.md#codepedent-resources)
+  by assigning the same flavor to codependent resources in a pod set.
+- Support [pod overhead](https://kubernetes.io/docs/concepts/scheduling-eviction/pod-overhead/)
+  in Workload pod sets.
+- Default requests to limits if requests are not set in a Workload pod set, to
```

Comment:
```suggestion
- Define default requests to limits if requests are not set in a Workload pod set, to
```

Reply:
Maybe Set is a more accurate verb:

```
Set requests to limits
```

wdyt?

Reply:
Changed to set and added a link to k8s reference doc.